implementations/rust/ockam/ockam_command/src/kafka/create.rs

```diff
@@ -23,10 +23,16 @@ use crate::{
     service::start::start_service_impl,
     terminal::OckamColor,
     util::{node_rpc, parsers::socket_addr_parser},
-    CommandGlobalOpts,
 };
+use crate::node::{get_node_name, initialize_node_if_default};
+use crate::util::process_nodes_multiaddr;
+
+#[derive(Clone, Debug, Args)]
+pub struct ArgOpts {
+    pub endpoint: String,
+    pub kafka_entity: String,
+}
 
-/// Create a new Kafka Producer
 #[derive(Clone, Debug, Args)]
 pub struct CreateCommand {
```

Comment:
We need a `CreateCommand` struct for both the producer and consumer commands so we can write their specific documentation. It's OK if we have to duplicate the commands arguments. The point here is to reuse the `rpc` function you have below.

That way, we'll have two separate commands with their own descriptions BUT with the same _internal_ logic.

Reply:
Thanks for the comments. Will do refactoring.